modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -0,0 +1,285 @@
+= schema_registry
+:type: input
+:status: beta
+:categories: ["Integration"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Reads schemas from a schema registry.
+
+Introduced in version 4.32.2.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+input:
+  label: ""
+  schema_registry:
+    url: "" # No default (required)
+    auto_replay_nacks: true
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+input:
+  label: ""
+  schema_registry:
+    url: "" # No default (required)
+    include_deleted: false
+    subject_filter: ""
+    tls:
+      enabled: false
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+    auto_replay_nacks: true
+```
+
+--
+======
+
+== Metadata
+
+The `schema_registry` input adds the following metadata fields to each message:
+
+```text
+- schema_registry_subject
+- schema_registry_version
+```
+
+You can access these metadata fields using
+xref:configuration:interpolation.adoc#bloblang-queries[function interpolation].
+
+
+
+== Example
+
```

Comment:
@Jeffail, I have made minor adjustments to field descriptions. Please could you check?

Reply:
LGTM